llvm/tools/llvm-exegesis/lib/BenchmarkCode.h

```diff
@@ -17,6 +17,11 @@
 namespace llvm {
 namespace exegesis {
 
+struct ScratchMemoryStore {
```

Comment:
Can you add docstring to this struct and its fields?

Reply:
Removed this structure because it is not used